neuralmonkey/learning_utils.py

```diff
@@ -277,13 +282,13 @@ def run_on_dataset(tf_manager: TensorFlowManager,
                                      compute_losses=contains_targets,
                                      batch_size=batch_size)
 
-    result_data_raw = {runner.output_series: result.outputs
-                       for runner, result in zip(runners, all_results)}
+    result_data = {runner.output_series: result.outputs
+                   for runner, result in zip(runners, all_results)}
 
     if postprocess is not None:
-        result_data = postprocess(dataset, result_data_raw)
-    else:
-        result_data = result_data_raw
+        for series_name, postprocessor in postprocess:
+            postprocessed = postprocessor(dataset, result_data)
+            result_data[series_name] = postprocessed
```

Comment:
timhle se přepíše (raw) series_name, co už byla v result_data, ne? dělá mi trochu problém orientovat se v tom, co se tu děje. přestože ten kód znám. možná za to může ta rýmička.